### PR TITLE
Select changed parts

### PR DESCRIPTION
**Pull request by ar4s**

### CODE REVIEW

File: src/ralph_assets/media/assets/js/parts.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+			var self = this;
+			if(this.observe_last) {
+				$('input', self.formset_selector)
+					.bind('keydown', function(event){
+						var val = $(this).val();
+						if(val !== '') {
+							$(this).closest(self.row_element).removeClass('empty');
+						}
+						if(event.keyCode === 13) {
+							event.preventDefault();
```
Comment (mkurek) [inline src/ralph_assets/media/assets/js/parts.js]: please consider this topic: http://stackoverflow.com/questions/1000597/event-preventdefault-function-not-working-in-ie - and check if it's working on IE

File: src/ralph_assets/templates/assets/parts/change_parts.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+{% block base_sidebar %}{% endblock %}
+
+{% block extra_js %}
+{{ block.super }}
+<script src="{{ STATIC_URL }}assets/js/parts.js"></script>
+{% endblock %}
+
+{% block content %}
+
+	{% if get_data %}<pre>GET: {{ get_data }}</pre>{% endif %}
```
Comment (andrzej-jankowski) [inline src/ralph_assets/templates/assets/parts/change_parts.html]: ?

File: src/ralph_assets/media/assets/js/parts.js [hunk 3]
```diff
@@ -0,0 +1,130 @@
+$(document).ready(function () {
+	'use strict';
+	var Formset;
+
+	Formset = (function() {
+		function Formset(formset_selector, prefix, row_element, observe_last) {
+			this.formset_selector = formset_selector;
```
Comment (quamilek) [inline src/ralph_assets/media/assets/js/parts.js]: Replace `tabs` to `spaces`.

File: src/ralph_assets/media/assets/js/parts.js [hunk 4]
```diff
@@ -0,0 +1,130 @@
+$(document).ready(function () {
```
Comment (quamilek) [inline src/ralph_assets/media/assets/js/parts.js]: Add some documentation for this js module.

Comment (ar4s) [inline src/ralph_assets/media/assets/js/parts.js]: done

**Pull request closed without merging**